package.json

```diff
@@ -117,10 +120,14 @@
     "husky": "8.0.1",
     "jest-canvas-mock": "2.4.0",
     "lint-staged": "13.1.0",
+    "msw": "^1.0.0",
```

Comment:
Should this be pinned like other dependencies?

Reply:
I pin it in my next PR but I'll pin it here too for good measure 👍